### PR TITLE
Fix build script, MNTR should output to TestResults/multinode

### PR DESCRIPTION
**Pull request by Arkatufus**
## Changes

MNTR tests were outputting their logs inside the test assembly folder, need to move them to the TestResults/multinode folder.

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -325,30 +325,36 @@ Target "MultiNodeTestsNetCore" (fun _ ->
...
-                | true -> (sprintf "test -c Release --no-build --logger:trx --logger:\"console;verbosity=normal\" --framework %s --results-directory \"%s\" -- -parallel none -teamcity" testNetCoreVersion outputMultiNode)
-                | false -> (sprintf "test -c Release --no-build --logger:trx --logger:\"console;verbosity=normal\" --framework %s --results-directory \"%s\" -- -parallel none" testNetCoreVersion outputMultiNode)
-
+                | true -> (sprintf "test \"%s\" -l:trx -l:\"console;verbosity=detailed\" --framework %s --results-directory \"%s\" -- -teamcity" projectDll testNetCoreVersion outputMultiNode)
+                | false -> (sprintf "test \"%s\" -l:trx -l:\"console;verbosity=detailed\" --framework %s --results-directory \"%s\"" projectDll testNetCoreVersion outputMultiNode)
+
+            let resultPath = (directory projectDll)
+            File.WriteAllText(
+                (resultPath @@ "xunit.multinode.runner.json"),
+                (sprintf "{\"outputDirectory\":\"%s\"}" outputMultiNode).Replace("\\", "\\\\"))
```
Comment (Arkatufus) [inline build.fsx:345]: Inject a .json setting file to set the MNTR test adapter output directory to "TestResults/multinode"

File: build.fsx [hunk 2]
```diff
@@ -325,30 +325,36 @@ Target "MultiNodeTestsNetCore" (fun _ ->
...
+            )
+        
+        let runSingleProject projectDll =
             let arguments =
                 match (hasTeamCity) with
-                | true -> (sprintf "test -c Release --no-build --logger:trx --logger:\"console;verbosity=normal\" --framework %s --results-directory \"%s\" -- -parallel none -teamcity" testNetCoreVersion outputMultiNode)
-                | false -> (sprintf "test -c Release --no-build --logger:trx --logger:\"console;verbosity=normal\" --framework %s --results-directory \"%s\" -- -parallel none" testNetCoreVersion outputMultiNode)
-
+                | true -> (sprintf "test \"%s\" -l:trx -l:\"console;verbosity=detailed\" --framework %s --results-directory \"%s\" -- -teamcity" projectDll testNetCoreVersion outputMultiNode)
+                | false -> (sprintf "test \"%s\" -l:trx -l:\"console;verbosity=detailed\" --framework %s --results-directory \"%s\"" projectDll testNetCoreVersion outputMultiNode)
```
Comment (Arkatufus) [inline build.fsx:340]: Targetting the test assembly directy, so we don't need the --no-build and -c Release argument anymore

Comment (Aaronontheweb) [reply build.fsx:340]: Got it - this was to work-around new `PWD` / log visibility issues introduced by `dotnet test`, which didn't occur when we were using our own custom binary before.

File: build.fsx (hunk 1 above)
Comment (Aaronontheweb) [reply build.fsx:345]: Because we have to feed our config into XUnit now - got it.

**Pull request merged**